gr-fec/python/fec/polar/polar_channel_construction

```diff
@@ -34,7 +34,7 @@ def setup_parser():
                       default='BEC')
     parser.add_argument("-b", "--blocksize", type=int, dest="block_size",
                       help="specify block size of polar code (default=16)", default=16)
-    parser.add_argument("-s", "--desgin-snr", type=float, dest="design_snr",
+    parser.add_argument("-s", "--design-snr", type=float, dest="design_snr",
```

Comment:
please review closely

Reply:
As long as dest doesn't change, it should just be a cosmetic thing, so looks good to me.